env.sample

```diff
@@ -1,40 +1,56 @@
 PROJECT_NAME=dead_songs
+DEBUG=True
```

Comment:
is there a reason not to have this line in the backend exemplar?
@MikeTheCanuck @nam20485

Reply:
It should be `True` for a local development build and `False` for a deploy / production build. I guess we can factor that out into the scripts that drive the builds / deploys.

Reply:
Should be fine for developer-only usage. I get more worked up when I see DEBUG enabled in the code that ships through Travis for any cloud deployment.

Reply:
The original intended workflow was:

1. Clone the repo.
2. Run the "create" script to initialize all the Django project files.
3. Push to a *new* repository for the project.
4. If you have a database, add its backup to "Backups" and do a "build".
5. Develop API code, add data to the database, test locally, push updates, etc.
6. When you're ready for "release", take a full backup and hand it off to DevOps.
7. After DevOps has restored the backup, deploy and debug against the real front end.

For Transportation Systems, we didn't do testing of the API against a local instance of the front end - we tested the API with browsers. I don't know what the other project teams did.